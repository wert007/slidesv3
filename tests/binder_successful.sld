func main() {
    let a = 1 + 1;
    print(a);
    let b = true != false;
    print(b);
    let c = (1 == 2) == (3 != 4);
    print(c);
    let d = 99;
    print(d);
    if 9 == 12 {
        let e = 14;
        print(e);
    } else {
        print('if condition was false.');
    }
    let f = 1 + 2 * 3;
    print(f);
    let g = 3 * 2 + 1;
    print(g);
}