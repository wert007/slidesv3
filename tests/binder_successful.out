2
true
false
99
if condition was false.
7
7
